deployment/enterprise/docker-compose.sso.yml

```diff
@@ -0,0 +1,17 @@
+networks:
+  mergin:
+    name: mergin-ee
+
+services:
+  jackson:
+    image: boxyhq/jackson:1.45.3
+    container_name: merginmaps-sso
+    restart: always
+    volumes:
+      - ./sso/sso-connections:/sso-connections
+    env_file:
+      - .sso.env
+    networks:
+      - mergin
+    ports:
```

Comment:
do we need this if they are on the same network as main stack?

Reply:
Should not be required. We can leave a comment here.